samples/net/sockets/mtls_http_get/server/client.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
I wonder if we should include these, or if we should just leave them out, and instruct people to run the generate script. We tend to get annoying vulnerability reports from people that scan github repos for keys. Realistically, though, I don't have a problem with these keys being in tree.